merge-v1/build/schema.graphql

```diff
@@ -1,44 +1,118 @@
-  rarityNow: BigDecimal! #TODO: see above comment. Same applies.
-  isBlue: Boolean!
-  isYellow: Boolean!
-  isNormal: Boolean! #TODO: Front end should be able to work with isBlue and isYellow to determine that it isNormal, but I'll put it in and see what DK says.
-  purchasePrice: BigDecimal # original purchase ether price of NFT
-  purchaseDate: BigDecimal #blockNumber converted to human time?
-  tier: BigInt!
-  class: BigInt!
-  mergeCount: BigInt!
-  tokenStandard: String!
+  "Unique tokenID for a respective NFT"
+  tokenID: BigInt!
+  "Size of mass for a specific NFT"
+  massSize: BigInt! #pak
+  "Color of mass NFT; BLACK is Alpha"
+  color: Color! #pak: class in smart contract specifies color where decodeClass(_values[tokenId]) is used in generating metadata (on-chain svg).
+  "Rarity for mass NFTs; higher numbers are highest rarity"
+  class: Class! #pak: in smart contracts, class is a uint
+  "Amount of times other NFTs within merge collection have been absorbed by this tokenID"
+  mergeCount: BigInt! #pak
+  "Accounts whitelisted to carry more than one mass NFT"
+  whitelist: Bool! #These are central marketplaces typically like Nifty Gateway
 }
 
-"These are the original merge nfts upon nft project initialization"
-type OriginalMasses @entity {
+"""
+Individual NFTs within this collection.
+"""
+type NFTs @entity {
+  "Unique tokenID"
   id: ID!
-  count: BigInt! #number of original separate mass nfts before "merging" was allowed and thus unique owners. From the get-go a user could not hold more than one "merge" nft cause it would just get merged into other merge nfts in that respective wallet
+  "Last sale ID showing number of times this token has been sold"
+  sale: [Sales!]! @derivedFrom(filed: tokenID)
+  "Whether the NFT has been merged, and therefore deleted"
+  merged: Boolean! #pak
+  "Current owner unless merged where it would be address(0)"
+  owner: Bytes!
+  "Size of mass for a specific NFT"
+  massSize: BigInt! #pak
+  "Color of mass NFT; BLACK is Alpha"
+  color: Color! #pak: class in smart contract specifies color where decodeClass(_values[tokenId]) is used in generating metadata (on-chain svg).
+  "Rarity for mass NFTs; higher numbers are highest rarity"
+  class: Class! #pak: in smart contracts, class is a uint
+  "Amount of times other NFTs within merge collection have been absorbed by this tokenID"
+  mergeCount: BigInt! #pak
 }
 
-"These are the current masses after the absorption of smaller masses in trades to date"
-type CurrentMasses @entity {
+"""
+The NFT collection, merge., and inherent global details on it.
+"""
+type Collection @entity {
+  "Set to 1"
   id: ID!
-  count: BigInt! #number of new separate mass nfts after "merging" allowed and thus unique owners
+  "Name of NFT collection"
+  collectionName: String!
```

Comment:
I would reduce this to just `name` because it is already obious this is the Collection. 

for example, someone querying this would have to write:

```
collection{
  collectionName
}
```